ladybug_comfort/et.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""Effective temperature transitioned from legacy code of @stgeorges"""
+
+def effective_temperature(Ta, ws, rh, SR, ac):
+    # inputs: (Ta, ws, rh, SR, ac):
+    if ws <= 0.2:
+        # formula by Missenard
+        TE = Ta - 0.4*(Ta - 10)*(1-rh/100)
+    elif ws > 0.2:
+        # modified formula by Gregorczuk (WMO, 1972; Hentschel, 1987)
+        TE = 37 - ( (37-Ta)/(0.68-(0.0014*rh)+(1/(1.76+1.4*(ws**0.75)))) ) - (0.29 * Ta * (1-0.01*rh))
+    
+    # Radiative-effective temperature
+    TRE = TE + ((1 - 0.01*ac)*SR) * ((0.0155 - 0.00025*TE) - (0.0043 - 0.00011*TE))
+    
+    if TRE < 1:
+        effectTE = -4
+        comfortable = 0
```

Comment:
This whole set of conditionals should be in a separate function. Much like you see here in the apparent temperature module:

https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/at.py

Also, we don't need the 0/1 for comfortable. We can infer that from the category.

Reply:
@chriswmackey im working on documentation at the moment and had a question, do you know what the variable effectTE is? I would like to document it